src/cdk/overlay/position/flexible-connected-position-strategy.spec.ts

```diff
@@ -1799,6 +1799,110 @@ describe('FlexibleConnectedPositionStrategy', () => {
     });
   });
 
+  describe('panel classes', () => {
```

Comment:
Add a test for not removing an existing css class on the element?

Reply:
Done.